core/storage/btree.rs

```diff
@@ -4653,8 +4785,9 @@ impl BTreeCursor {
                     match cell {
                         //  For an index interior cell, clear the left child page now that overflow pages have been cleared
                         BTreeCell::IndexInteriorCell(index_int_cell) => {
-                            let (child_page, c) =
-                                self.read_page(index_int_cell.left_child_page as i64)?;
+                            let (child_page, c) = return_if_io!(
```

Comment:
How does this work? If we yield on IO here, then `child_page` won't be pushed to stack. If we don't yield on IO here, then is `c` ever `Some`?